raiden/tests/smart_contracts/test_netting_channel.py

```diff
@@ -387,6 +387,21 @@ def test_close_called_multiple_times(tester_state, tester_nettingcontracts):
     assert nettingchannel.closingAddress(sender=pkey0) == encode_hex(address0)
 
 
+def test_close_tampered_nonce(tester_state, tester_channels):
+    """ Messages the nonce tampered must be rejected. """
+    pkey0, pkey1, nettingchannel, channel0, channel1 = tester_channels[0]
+
+    transfer0 = make_direct_transfer_from_channel(channel0, channel1, amount=90, pkey=pkey0)
+    transfer0_data = transfer0.encode()
+
+    tampered_transfer = DirectTransfer.decode(transfer0_data)
+    tampered_transfer.identifier += 1
```

Comment:
The test is named `tampered_nonce` but here you are messing with the identifier. Typo?